src/indexer/index_writer.rs

```diff
@@ -1401,15 +1434,41 @@ mod tests {
                     .map(move |doc| ff_reader.get(doc))
             })
             .collect();
+        ids.sort();
```

Comment:
we should probably disable the sorted is not enabled? (that requires disabling it in the expected_ids function too)

Reply:
the sort is required because the order is not kept with the merges. I guess a set would be better suited?

Reply:
ah I see. I guess that's ok.